test/asammdf/gui/widgets/test_PlotWidget_Shortcuts.py

```diff
@@ -231,3 +236,264 @@ def test_Plot_Plot_Shortcut_Key_R(self):
         # Save PixMap of clear plot
         clear_pixmap = plot.plot.viewport().grab()
         self.assertTrue(Pixmap.is_black(clear_pixmap))
+
+    def test_Plot_Plot_Shortcut_Key_F(self):
+        """
+        Test Scope:
+            Check if all plotted signals is fitted on Y ax after pressing key "F".
+        Events:
+            - Open 'FileWidget' with valid measurement.
+            - Select 2 measurement and create a plot
+            - Press Key "S"
```

Comment:
Maybe one extra detail needs to be written here.
The precondition is that channels should not be streched on entire plot heigth.
'S' - 'Stack' command is used to ensure that channels are resized to fulfill the precondition.